src/main/java/uk/gov/companieshouse/pscfiling/api/service/impl/CompanyProfileServiceImpl.java

```diff
@@ -11,6 +11,9 @@
 import uk.gov.companieshouse.pscfiling.api.service.CompanyProfileService;
 import uk.gov.companieshouse.pscfiling.api.utils.LogHelper;
 
+/**
+ * The company profile service layer.
+ */
```

Comment:
I'm not sure, but do we need to duplicate the same comment in the implementation as is in the interface?

Reply:
Agreed, removed Impl javadoc if already present in interface.